notebooks/ICESat-2_MODIS_Arctic_Sea_Ice/Customize and Access Data.ipynb

```diff
@@ -426,7 +427,7 @@
     "    # Create session to store cookie and pass credentials to capabilities url\n",
     "    session = requests.session() \n",
     "    s = session.get(capability_url)\n",
-    "    response = session.get(s.url,auth=(uid,pswd))\n",
+    "    response = session.get(s.url, headers={\"Authorization\": f\"Bearer {auth}\"})\n",
```

Comment:
```suggestion
    "",
```

Reply:
It appears that this line can go away entirely because of the earthaccess HTTPS session method